### DISCUSSION TITLE
command palette

### DISCUSSION THREAD

**Discussion by Dan-Patterson** [community/community#15236] [Other Feature Feedback, Questions, & Ideas]
ctrl-k  
fine for those that want to memorize keyboard shortcuts for all the applications they use and those that don't conflict with other applications.
For me? If I don't use it all the time, or I can't see it, I probably won't remember it, so in a few weeks I will probably won't even remember "command palette"

**Comment by alxfed** [comment]
useless.

**Comment by multimokia** [comment]
I already have a keyboard shortcut mapped to ctrl+k (powertoys search), personally would rather be able to manually configure the shortcuts or flat out disable them as in the case of the command pallete, I just don't find it useful vs standard mouse navigation.

**Comment by queenofcorgis** [comment]
Hi there @Dan-Patterson 👋🏼  Thanks for the feedback, we have a feature that allows you to customize the keyboard shortcuts used to open the command palette. You can edit your shortcuts in the [Accessibility section](https://github.com/settings/accessibility) of your user settings.

**Reply by Dan-Patterson** [reply]
how about an icon? menu? keyboard shortcuts should be a supplement for, and not a replacement to functionality access

**Reply by queenofcorgis** [reply]
@Dan-Patterson thanks for sharing your perspective, I've unmarked this as answered so our team can review this feedback!

**Comment by qupufu** [comment]
Would be nice if you could open your stars somehow. Other than that, I've found it really nice.

**Reply by queenofcorgis** [reply]
Would you mind elaborating on this request please? Are you interested in a command that opens your starred repositories or something else?

**Reply by qupufu** [reply]
Yeah to open your starred repositories

**Reply by raf181** [reply]
yea i think that option would be cool since it remuves the need for searching theme fom the stars section and its faster

**Comment by lohanidamodar** [comment]
How can I search for files in specific branch? I always get redirected back to default branch while searching and navigating to file from command palette?

**Comment by raf181** [comment]
useful for the people that do not have keyboard shortcuts but for easy setings changes or pull recuests with out having to change page

**Comment by Pouya-moh** [comment]
I hate that `ctrl-k` gets hijacked. It is, and has been, the default shortcut of Firefox's jump to search bar. Making assumption that people use address bar for searching (`ctrl-l`) hence nobody needs `ctrl-k` is not cool.

**Reply by ChristianTacke** [reply]
Have you tried https://github.com/settings/accessibility to change the keyboard bindings?
